pkg/ovsdb/transact.go

```diff
@@ -1020,8 +1023,8 @@ func doSelect(txn *Transaction, ovsOp *libovsdb.Operation, ovsResult *libovsdb.O
 		if err != nil {
```

Comment:
1. we have to set error in the operation result,
2.  we have to clean Row there

Reply:
ok